modules/grafana/files/dashboards/sidekiq.json

```diff
@@ -71,20 +50,20 @@
           "steppedLine": false,
           "targets": [
             {
+              "hide": false,
               "refId": "A",
-              "target": "aliasByNode(averageSeriesWithWildcards(stats.gauges.govuk.app.$Application.*.workers.queues.$Queues.enqueued, max, 5), 7)",
+              "target": "aliasByNode(summarize(averageSeriesWithWildcards(stats_counts.govuk.app.$Application.*.workers.*.success, 4), '1min', 'sum', false), 5)",
```

Comment:
I think more information needs to be given about what this panel is showing, either in the panel title, or on the y axis.

My interpretation of this query is that it shows the average number of jobs processed in a minute, per machine.

Reply:
Good spot! That's me getting the query wrong - I've changed it to a summation, which should match the intention of the title. Does that help?

Reply:
Yep, changing it to a summation is good.

Additionally, I think that specifying a label for the y axis would be good, maybe "Jobs per minute" or "Processed per minute", or something else to tell someone what the numbers mean, without having to look at the Graphite query.